config.json

```diff
@@ -888,9 +888,22 @@
       "uuid": "6b7f7b1f-c388-4f4c-b924-84535cc5e1a0",
       "slug": "hexadecimal",
       "deprecated": true
+    },
```

Comment:
It would probably be best to move the `config.json` entry for this exercise before all deprecated exercises.